### PR TITLE
rmw serialized to rcutils serialized

### PR DESCRIPTION
**Pull request by Karsten1987**
fixes https://github.com/ros2/rmw/issues/144

I let the original test for the serialized message in this package to ensure the typedefs working.
I am happy to remove them if considered as unnecessary.

### CODE REVIEW

File: rmw/test/test_serialized_message.cpp [hunk 1]
```diff
@@ -16,7 +16,7 @@
 
 #include "rcutils/allocator.h"
 
-#include "rmw/serialized_message.h"
+#include "rmw/types.h"
```
Comment (mikaelarguedas) [inline rmw/test/test_serialized_message.cpp]: Nit: Is this change necessary ?

Comment (Karsten1987) [reply rmw/test/test_serialized_message.cpp]: I need that for the return values.
